plugins/performance-lab/includes/admin/plugins.php

```diff
@@ -114,26 +113,25 @@ function perflab_render_plugins_ui(): void {
 				),
 				array( 'type' => 'error' )
 			);
-			continue;
```

Comment:
Why is this removed? Won't this cause errors down the line if we pass a `WP_Error` to `array_merge()`?

Reply:
No. There is a `if ( $api_data instanceof WP_Error ) {` safeguard, so it's always an array when merging. If not, PHPStan would complain.

For context, I refactored it this way in #1098 so that the PL modules page still works if the plugin is not yet in WordPress.org (like when developing locally). Otherwise you'd get  a`Failed to query WordPress.org Plugin Directory for plugin "web-worker-offloading". Plugin not found.` notice right now, which doesn't make sense.

Reply:
@swissspidy Oh, what I had missed due to the diff was that now the entire code below is in an `else` for this `if`. That's why I was confused about the missing `continue`. :)